test-node-version.sh

```diff
@@ -0,0 +1,14 @@
+VOLUMES="-v $(pwd)/index.js:/test/index.js -v $(pwd)/index.test.js:/test/index.test.js -v $(pwd)/package.json:/test/package.json -v $(pwd)/jest.config.js:/test/jest.config.js"
```

Comment:
:+1:  Kudos for testing different node versions!

Reply:
thanks :-)